logstage/logstage-api/src/main/scala/izumi/logstage/macros/LogMessageMacro.scala

```diff
@@ -148,6 +148,11 @@ object LogMessageMacro {
         val namedArgs = ArgumentNameExtractionMacro.recoverArgNames(c)(args.map(p => c.Expr(p)))
         reifyContext(c)(stringContext, namedArgs)
 
+      case Select(Apply(_, List(Apply(Select(stringContext@Apply(Select(Select(Ident(TermName("scala")), TermName("StringContext")), TermName("apply")), _), TermName("s")), args: List[c.Tree]))), TermName("stripMargin")) =>
+        val namedArgs = ArgumentNameExtractionMacro.recoverArgNames(c)(args.map(p => c.Expr(p)))
+        val sc = q"""StringContext($stringContext.parts.map(_.stripMargin): _*)"""
```

Comment:
Better `_root_.scala.StringContext`, so that it won't break if user has a local variable named `StringContext`.

Reply:
Done.